extensions-compose/src/commonMain/kotlin/com/arkivanov/decompose/extensions/compose/stack/Children.kt

```diff
@@ -25,7 +25,7 @@ fun <C : Any, T : Any> Children(
 
     holder.retainStates(stack.getConfigurations())
 
-    val anim = animation ?: emptyStackAnimation()
+    val anim = animation ?: LocalStackAnimation.current.invoke()
```

Comment:
Currently (without this change) the logic is as follows:

1. If `animation` argument is explicitly supplied as non-null then the supplied `StackAnimation` is used.
2. If `animation` argument is explicitly supplied as null or not supplied then `emptyStackAnimation` is applied..

Let's keep this behaviour here as well for consistency (i.e. `null` means default). I think it should be as follows.

1. If `animation` argument is explicitly supplied as non-null, then the supplied `StackAnimation` is used
2. If `animation` argument is explicitly supplied as null or not supplied
    * If `LocalStackAnimationProvider` returns a non-null `StackAnimation`, then the returned `StackAnimation` is used
    * Otherwise `emptyStackAnimation` is used

I suggest it should be something like this:

```kotlin
    val animationProvider = LocalStackAnimationProvider.current
    val anim = animation ?: remember(animationProvider, animationProvider::provide) ?: emptyStackAnimation()
```

This will allow resetting `LocalStackAnimationProvider` to whatever default animation is used by Decompose (i.e. `emptyStackAnimation`).